_episodes/01-introduction.md

```diff
@@ -3,12 +3,202 @@ title: "Introduction"
+* Basic computer vision tasks such as edge detection, decreasing colour depth or blurring an image.
+
+
+> ## Application areas/History
+>
+> some benchmarks
+> DL is not new as a concept but availability of comp resources is now at stage that it can be done efficiently
+> some examples of interesting things that can be achieved using DL models:
+> image classification, text generation, language translation, GAN, Gameplay (Dota/Go), voice recognition,
+> self-driving cars, natural language and image processing, predictive forecasting, fraud detection in financial applications,
+> financial time-series forecasting, predictive and prescriptive analytics, medical image processing, power systems research,
+> recommendation systems
+> we are still far from computers understanding us
+>
+{: . callout}
+
+> ## What is Deep Learning:
+>
+> Give multiple definitions of Deep Learning and discuss;
+> OR
+> show both bubble representations of AI-ML-NN-DL and AI-ML-DL, ask which is right
+>
+{: .discussion}
+
+
+> ## Deep Learning Problems Exercise
+> Which of the following could you apply deep learning to?
+> 1. Recognising whether or not a picture contains a bird.
+> 2. Calculating the median and interquartile range of a dataset.
+> 3. Identifying pictures of people when only one or two images of them are available.
+> 4. Identifying people in pictures after being trained only on cats and dogs.
+> 5. Translating English into French.
+>> ## Solution
+>> 1 and 5 are the sort of tasks often solved with deep learning. 2. technically possible but deep learning is massively overkill, you could do the same with much less computing power using traditional techniques. 3. will fail because there's not enough training data. 4. will fail because the deep learning system only knows what cats and dogs look like, it might accidentally classify the people as cats or dogs.
+> {: .solution}
+{: .challenge}
+
+> ## What is DL - solution
+>
+> learning outcome -> it is vague, find your own
+> OR
+> hah! both are right in some ways, explain why NN has its place there as well
+>
+{: .solution}
+
+*the neuron/perceptron/unit/
+
+* inputs, weight, sum, non-linearity, output
+* -> DL: stacked perceptrons
+
+* training challenges
+  * gradient
+  * overfitting
+
+## Building blocks of NN
+
+*building blocks of a DL model:
+    *example case CNN
+        *show how each layer may look
+        *eg based on face features
+        *we learn from many examples what and eye looks like and where it is located
+*-> needs high amount of data
+
+## Deep Learning workflow
+
+(one way, open for discussion and to be filled with information)
+1. Formulate/ Outline the problem
+
+2. Identify inputs and outputs
+
+3. Prepare data
+common steps for data preparation
+
+4. Choose a cost function and metrics
+
+5. Choose a pretrained model or start building architecture from scratch
+build from scratch vs reuse vs transfer learning
+
+6. Train model
+
+7. Tune hyperparameters
+
+8. 'predict'
```

Comment:
Before, or perhaps instead of, predict I would add a section on measuring performance. This would neatly align the tuning of hyper parameters with a validation set and performance measurement with a test set.